setup.py

```diff
@@ -36,7 +36,7 @@ def run(self):
 
 
 setup(name='gemd',
-      version='0.6.6',
+      version='0.6.7',
```

Comment:
Minor version bump because it's a new feature?

Reply:
The deprecation warnings on the Taurus migration are:
```
./gemd/client/json_encoder.py:@deprecation.deprecated(deprecated_in="0.6.0", removed_in="0.7.0",
```
So, from a process perspective, I should be wiping out the Taurus tree in a different PR if we're doing the minor version bump.

Reply:
ack

Well I don't know what the best process is... but this PR itself looks fine.